examples/particle/particle.rs

```diff
@@ -198,7 +202,7 @@ impl System {
         }
     }
 
-    pub fn draw(&self, pass: &mut gpu::RenderCommandEncoder) {
+    pub fn draw(&self, pass: &mut gpu::RenderCommandEncoder, size: (u32, u32)) {
```

Comment:
should we just get the aspect here?

Reply:
Used the size and sent it directly to the shader before but changed it to be the aspect ratio xP Maybe should be the input to the function now yeah since that is what is used